src/node/node_class.hpp

```diff
@@ -266,7 +266,7 @@ static inline std::vector<napi_value> napi_get_arguments(const Napi::CallbackInf
 }
 
 template<typename ClassType>
-inline ClassType& ObjectWrap<ClassType>::get_class() {
+inline auto& ObjectWrap<ClassType>::get_class() {
```

Comment:
I am a bit curious: why change to `auto`?

Reply:
mostly for consistency. See this comment https://github.com/realm/realm-js/pull/3434#discussion_r544312411